src/app/QuickStarts/quickstarts/topology/custom-target-quickstart.tsx

```diff
@@ -75,28 +75,28 @@ However, in some cases (e.g. for **Kubernetes API**, JMX port is not <code>9091<
         description: `
 1. Enter the JMX **Connection URL** for the target into the [Connection URL]{{highlight ct-connecturl-input}} field.
 2. Optionally assign an **Alias** to the target by using the [Alias]{{highlight ct-alias-input}} field.
-3. If the target has **JMX authentication** enabled, click the [JMX Credential Option]{{highlight ct-credential-expand}} to expand the form.
+3. If the target has **JMX authentication** enabled, click the [JMX credential option]{{highlight ct-credential-expand}} to expand the form.
```

Comment:
I think `JMX Credential option` might make sense, since (Stored) Credentials are also a kind of resource